addons/html_editor/static/src/core/line_break_plugin.js

```diff
@@ -63,6 +66,13 @@ export class LineBreakPlugin extends Plugin {
         const brEls = [brEl];
         if (targetOffset >= targetNode.childNodes.length) {
             targetNode.appendChild(brEl);
+            if (
+                !isBlock(closestElement(targetNode)) &&
+                nextLeaf(targetNode, closestBlock(targetNode))
+            ) {
+                targetNode.appendChild(this.document.createTextNode("\u200B"));
+                targetNode.setAttribute("data-oe-inline-line-break", "");
```

Comment:
Why not use "\ufeff" instead? If I'm not mistaken the system already removes them all, without requiring a new special "data-oe" attribute to mark them.

Reply:
It will not work because, when normalize is called, it will remove those '\ufeff' characters.